xpsi/PostProcessing/_runs.py

```diff
@@ -122,10 +125,57 @@ def _transform(q, **kwargs):
             _transform = None
             _overwrite = False
 
+        if multi_mode:
+            for vec in range(len(roots)):
+                filerootpath =_os.path.join(base_dirs[vec], roots[vec])
+                # Read the file and manually handle blank lines
+                with open(filerootpath+"post_separate.dat", 'r') as file:
+                    lines = file.readlines()
+
+                # Convert lines to a numpy array, replacing blank lines with a marker (e.g., None or NaN)
+                data = []
+                for line in lines:
+                    stripped = line.strip()
+                    if stripped:  # Non-blank line
+                        data.append(_np.array(stripped.split(), dtype=float))
+                    else:  # Blank line
+                        data.append(None)
+
+                # Separate modes based on None markers
+                modes = []
+                current_mode = []
+                for row in data:
+                    if row is None:
+                        if current_mode:
+                            modes.append(_np.array(current_mode))
+                            current_mode = []
+                    else:
+                        current_mode.append(row)
+
+                # Append the last mode if not empty
+                if current_mode:
+                    modes.append(_np.array(current_mode))
+
+                for mode in range(len(modes)):
+                    _np.savetxt(filerootpath+f"mode{mode+1}.txt", modes[mode])
+                    roots.append(roots[vec]+f"mode{mode+1}")
+                    run_IDs.append(run_IDs[vec]+f"{mode_label} {mode+1}")
+                    base_dirs.append(base_dirs[vec])
+                    use_nestcheck.append(use_nestcheck[vec])
+            # Forget about the defaul xpsi loaded file
```

Comment:
I suggest to remove these lines so the user can still load the total run if needed